templates/blast-form.html

```diff
@@ -6,6 +6,15 @@
 <script src="{{ url_for('static', filename='js/blast.js') }}"></script>
 {% endblock %}
 
+{% block og_meta %}
+  <meta property="og:url" content="https://support.texastribune.org/blastform">
+  <meta property="og:image" content="https://static.texastribune.org/media/membership/tt-blast-share.png">
```

Comment:
**question:** Would it make sense to store this at the repo level?  We seem to do that for the social image [on the main page](https://github.com/texastribune/donations/blob/ad53ecc8f4d031c800b23b4ca6ccd7d786933f78/templates/new_layout.html#L15).

Reply:
Ah didn't notice that, great suggestion! :done: